pkg/cluster/member.go

```diff
@@ -38,6 +39,11 @@ func (c *Cluster) updateMembers(known etcdutil.MemberSet) error {
 				return errUnexpectedUnreadyMember
 			}
 
+			if !strings.HasPrefix(m.Name, c.cluster.Metadata.GetName()) {
```

Comment:
right. we need to compare the client url against the seed url. if it is one of the seed, we need to ignore it.